blog/util/request_parser.py

```
@@ -0,0 +1,6 @@
+def get_query_string_filter(request, *args) -> dict[str, str]:
```

Ruff (0.15.5)

[warning] Missing type annotation for `*args`

(ANN002)